common/config/src/lib.rs

```diff
@@ -298,6 +312,25 @@ pub fn get_hostname() -> Option<String> {
     System::new_with_specifics(RefreshKind::new()).get_host_name()
 }
 
+fn print_settings(yaml: &str, config_path: &Path) {
+    print!("Listing current settings ");
+
+    let is_default_path = config_path.to_string_lossy() == argv::DEFAULT_YAML_FILE;
+
+    if config_path.exists() {
+        print!("from config ({}), ", config_path.display());
+    } else if is_default_path && Path::new(argv::DEFAULT_CONF_FILE).exists() {
```

Comment:
Is it necessary to differentiate between the default config file and non-default config file here? Either way it's just "options from the config file"

Reply:
I think it can be useful for troubleshooting. The agent tries to read from the `/etc/logdna.conf` when the default `/etc/logdna/config.yaml` is not found and it can be surprising.